website/docs/intersection-types.md

```diff
@@ -28,10 +28,10 @@ for intersection types:
 As the names indicate, union types and intersection types can be understood by
 analogy to the related operations on sets. Given `Type1` and `Type2`:
 
-- `T.any(Type1, Type2)` is the union of the set of values that either have type
-  `Type1` or have type `Type2`.
```

Comment:
I also felt that ``have type `Type2` `` doesn't flow well when spoken out loud — I swapped "have type TypeX" with "are of TypeX" to try to help with that